pkg/daemon/daemon_test.go

```diff
@@ -1,313 +1,316 @@
-				ObjectMeta: metav1.ObjectMeta{
-					Name:       "test-node",
-					Generation: 777,
-				},
-			}
-			Expect(
-				updateSriovNetworkNodeState(sut.client, nodeState2)).
-				To(BeNil())
-
-			var msg Message
-			Eventually(refreshCh, "10s").Should(Receive(&msg))
-			Expect(msg.syncStatus).To(Equal("InProgress"))
-
-			Eventually(refreshCh, "10s").Should(Receive(&msg))
-			Expect(msg.syncStatus).To(Equal("Succeeded"))
-
-			Expect(sut.nodeState.GetGeneration()).To(BeNumerically("==", 777))
-		})
-	})
-
-	Context("isNodeDraining", func() {
-
-		It("for a non-Openshift cluster", func() {
-			sut.openshiftContext = &utils.OpenshiftContext{IsOpenShiftCluster: false}
-
-			sut.node = &corev1.Node{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:        "test-node",
-					Annotations: map[string]string{}}}
-
-			Expect(sut.isNodeDraining()).To(BeFalse())
-
-			sut.node.Annotations["sriovnetwork.openshift.io/state"] = "Draining"
-			Expect(sut.isNodeDraining()).To(BeTrue())
-
-			sut.node.Annotations["sriovnetwork.openshift.io/state"] = "Draining_MCP_Paused"
-			Expect(sut.isNodeDraining()).To(BeTrue())
-		})
-
-		It("for an Openshift cluster", func() {
-			sut.openshiftContext = &utils.OpenshiftContext{
-				IsOpenShiftCluster: true,
-				OpenshiftFlavor:    utils.OpenshiftFlavorDefault,
-			}
-
-			sut.node = &corev1.Node{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:        "test-node",
-					Annotations: map[string]string{}}}
-
-			Expect(sut.isNodeDraining()).To(BeFalse())
-
-			sut.node.Annotations["sriovnetwork.openshift.io/state"] = "Draining"
-			Expect(sut.isNodeDraining()).To(BeTrue())
-
-			sut.node.Annotations["sriovnetwork.openshift.io/state"] = "Draining_MCP_Paused"
-			Expect(sut.isNodeDraining()).To(BeTrue())
-		})
-
-		It("for an Openshift Hypershift cluster", func() {
-			sut.openshiftContext = &utils.OpenshiftContext{
-				IsOpenShiftCluster: true,
-				OpenshiftFlavor:    utils.OpenshiftFlavorHypershift,
-			}
-
-			sut.node = &corev1.Node{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:        "test-node",
-					Annotations: map[string]string{}}}
-
-			Expect(sut.isNodeDraining()).To(BeFalse())
-
-			sut.node.Annotations["sriovnetwork.openshift.io/state"] = "Draining"
-			Expect(sut.isNodeDraining()).To(BeTrue())
-
-			sut.node.Annotations["sriovnetwork.openshift.io/state"] = "Draining_MCP_Paused"
-			Expect(sut.isNodeDraining()).To(BeTrue())
-		})
-	})
-})
-
-func createSriovNetworkNodeState(c snclientset.Interface, nodeState *sriovnetworkv1.SriovNetworkNodeState) error {
-	_, err := c.SriovnetworkV1().
-		SriovNetworkNodeStates(namespace).
-		Create(context.Background(), nodeState, metav1.CreateOptions{})
-	return err
-}
-
-func updateSriovNetworkNodeState(c snclientset.Interface, nodeState *sriovnetworkv1.SriovNetworkNodeState) error {
-	_, err := c.SriovnetworkV1().
-		SriovNetworkNodeStates(namespace).
-		Update(context.Background(), nodeState, metav1.UpdateOptions{})
-	return err
-}
+//
```

Comment:
why do we have this commented out ? (if its handled in subsequent commit, disregard the comment)

Reply:
yep sorry I fix it :P